Frontend/ui-customer/src/Screens/TicketBookingFlow/TicketStatus/ScreenData.purs

```diff
@@ -66,7 +66,8 @@ initData =
 
 dummyData :: TicketBookings
 dummyData = { booked : [{shortId : "kjdfk;a", ticketPlaceName : "Zoological Garden, AliPore", amount : 500.0, visitDate : "10-10-2023", status : Booked, ticketPlaceId : "", personId : "" }],
-              pendingBooking : [{shortId : "kjdfadfk;a", ticketPlaceName : "Zoological Garden, AliPore", amount : 500.0, visitDate : "10-10-2023", status : Pending, ticketPlaceId : "", personId : ""}]
+              pendingBooking : [{shortId : "kjdfadfk;a", ticketPlaceName : "Zoological Garden, AliPore", amount : 500.0, visitDate : "10-10-2023", status : Pending, ticketPlaceId : "", personId : ""}],
+              cancelled : []
             }
```

Comment:
can we keep default as empty array for all the fields ? In `TicketList` and other screens as well